### PR TITLE
Adding optional monthly checkbox to 2016 eoy fundraising snippet.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@glogiotatidis review?

We'll probably want to run tests for this to see if we can prove some theories.

This one is to try a monthly checkbox.

Also see: https://github.com/mozmar/snippets/pull/90

**Comment by ScottDowne** [comment]
@glogiotatidis Thanks, updated.

**Comment by glogiotatidis** [comment]
Thanks Scott!

File: campaigns/mofo-eoy-2016/snippet.html [hunk 1]
```diff
@@ -290,15 +309,36 @@
             }
             btnSelected = element;
             amount = element.attributes["data-amount"].value;
-            btnDonate.attributes["href"].value = donationURL + delimiter + "amount=" + amount + "&currency=" + currencyCode + "&presets=" + presets;
           }
           var selected = "{{ selected_button|default('donation-amount-second', true) }}";
           selectAmount(snippet.querySelector("." + selected));
 
+          var frequency = "single";
+          {% if show_donation_amount_buttons and show_monthly_checkbox %}
```
Comment (glogiotatidis) [inline campaigns/mofo-eoy-2016/snippet.html]: Ah this was in `show_donation_amount_buttons` check anyway. Sorry for that

**Comment by glogiotatidis** [comment]
Pushed to production

**Pull request merged**